src/browser.ts

```diff
@@ -19,26 +19,27 @@ import { Page } from './page';
 import { EventEmitter } from 'events';
 import { Download } from './download';
 import { debugProtocol } from './transport';
+import type { BrowserServer } from './server/browserServer';
```

Comment:
This syntax did not compile for me locally. Did we bump ts version or something?

Reply:
yes